test/flink_test.go

```diff
@@ -10,6 +10,10 @@ func (s *CLITestSuite) TestFlinkHelp() {
 		{args: "flink compute-pool list -h", fixture: "flink/compute-pool/list-help.golden"},
 		{args: "flink compute-pool update -h", fixture: "flink/compute-pool/update-help.golden"},
 		{args: "flink compute-pool use -h", fixture: "flink/compute-pool/use-help.golden"},
+		{args: "flink iam-binding -h", fixture: "flink/iam-binding/help.golden"},
```

Comment:
@MuweiHe @sgagniere Can you take a look at #1958? It would be nice if contributors don't have to write their own help tests.

Reply:
Taking a look